shared/chat/conversation/messages/coinflip/errors.js

```diff
@@ -58,10 +78,55 @@ const CoinFlipAbsenteeError = (props: AbsenteeProps) => {
   )
 }
 
-const styles = Styles.styleSheetCreate({
-  error: {
-    color: Styles.globalColors.red,
-  },
-})
+const CoinFlipTimeoutError = () => {
```

Comment:
when just returning we prefer
```
const CoinFlipTimeoutError = () => <Kb.Text ...
```

Reply:
Why? It's just more annoying if you need to change it later, and doesn't really make the code easier to read or anything else.

Reply:
its a style choice thats applied everywhere. it does reduce how much noise there is but ultimately its a stylistic choice. what I'd like to avoid is having code that is against the style, which i think is worse than one style or the other